kolibri/plugins/learn/assets/src/views/action-bar-search-box/index.vue

```diff
@@ -0,0 +1,116 @@
+<template>
+
+  <div class="search-box-wrapper">
+    <div v-show="showSearchBox" :class="{ 'search-box-dropdown': showDropdownSearchBox }">
+      <search-box
+        ref="searchBox"
+        :icon="showDropdownSearchBox ? 'arrow_forward' : 'search'"
+        :width="showDropdownSearchBox ? 'calc(100vw - 120px)' : '150px'"
+      />
+    </div>
+
+    <ui-icon-button
+      v-show="showDropdownSearchBox"
+      icon="search"
+      type="primary"
+      color="primary"
+      @click="toggleDropdownSearchBox"
+    />
+
+    <div
```

Comment:
The reasoning behind that is so that the user focuses on search. Notice the mockups. https://projects.invisionapp.com/share/6HA34NCQM#/screens/226784437